VultisigApp/VultisigApp/Services/Solana/SolanaService.swift

```diff
@@ -8,6 +9,12 @@ class SolanaService {
     private let rpcURL = URL(string: Endpoint.solanaServiceRpc)!
     private let jsonDecoder = JSONDecoder()
     
+    private var cacheTokenAccountsByOwner: ThreadSafeDictionary<String, (data: [SolanaTokenAccount], timestamp: Date)> = ThreadSafeDictionary()
```

Comment:
lets stop using dictionary cache? It's slow because of sync with .barrier and not convenient to use imo. We can use URL cache. Just setup requestPolicy for request you need to cache

Reply:
Ok, I've created a Cache Service to handle that. We will use the ```CachedURLResponse```